samcli/lib/package/stream_cursor_utils.py

```diff
@@ -1,11 +1,19 @@
 """
 Stream cursor utilities for moving cursor in the terminal.
 """
+import os
 
 # NOTE: ANSI escape codes.
 # NOTE: Still needs investigation on non terminal environments.
 ESC = "\u001B["
 
+# Enables ANSI escape codes on Windows
+if os.name == "nt":
```

Comment:
We usually use `platform.system().lower() == "windows"` for this, any reason for the change?

Reply:
The initial thought was that `os.name` and `platform.system().lower()` are the same for this use case and `os` module is already imported for `os.system` call. However, I'm in favor of keeping the consistency with rest of the code. Updated to use `platform` instead.